provider/manifest/handler.go

```diff
@@ -218,12 +226,15 @@ func (h *handler) checkManifestState(ctx context.Context, mstate *manifestState,
 		// TODO: validate manifest
 
 		// publish complete manifest
-		h.bus.Publish(event.ManifestReceived{
-			LeaseID:    *mstate.leaseID,
-			Manifest:   mstate.request.Manifest,
-			Deployment: mstate.deployment,
-		})
-		h.session.Log().Debug("manifest complete", "deployment", did.EncodeString())
+		for _, lev := range mstate.leases {
+			h.bus.Publish(event.ManifestReceived{
```

Comment:
Error return value of `h.bus.Publish` is not checked

Reply:
this is cool